resources/articles/oauth/token.md

```diff
@@ -0,0 +1,199 @@
+:title POST /oauth/token
```

Comment:
This page should probably have a disclaimer about required client authentication (and link to "SPiD Authentication")

Reply:
🇦🇺